design/reboot-interface.md

```diff
@@ -89,22 +89,34 @@ section of the BareMetalHost status. This records a time after which the Host
 was last booted using the current image. Processes running prior to this time
 may be assumed to have been stopped.
 
-A new date-time field, ``lastReboot``, will be added to the ``provisioning``
+A new date-time field, ``lastRebootRequest``, will be added to the ``provisioning``
 section of the BareMetalHost status. This records a time after which the Host
-was last requested to reboot.
+was last requested to reboot (because we cannot trust any value from the user,
+who even if well intentioned, may have created the timestamp on a machine that 
+was not synchronised with the cluster or has a different timezone).
 
 Since the user interface requirements are still unclear, we will follow 
 standard practices of using an annotation ( ``host.metal3.io/reboot`` ) to 
```

Comment:
Would `fence` (or something else) be more accurate than `reboot` now that we're actually keeping power off until the annotation is removed?

Reply:
i would very much like to avoid the word _fence_ if possible,  and reboot is too overloaded as a term, so it should be more like power cycle.

Maybe something like: 
- staged-power-cycle
- ordered-power-cycle
- power-cycle-with-wait
- power-off-and-hold
- emergency-power-off
- power-fence